lib/components/narrative/default/default-itinerary.js

```diff
@@ -166,7 +166,7 @@ const ITINERARY_ATTRIBUTES = [
             includeSeconds={false}
           />
           <LegIconWrapper noSpace>
-            <LegIcon leg={leg} size={5} />
+            <LegIcon leg={{ ...leg, mode: 'WALK' }} size={5} />
```

Comment:
does legIcon even need the whole leg?

Reply:
yeah it complains if the start/end stuff is missing